### PR TITLE
Added response modification support

### PR DESCRIPTION
**Pull request by andykuszyk-form3**
Addresses https://github.com/form3tech/swift-gateway/issues/154

### CODE REVIEW

File: internal/app/pacts/myconsumer-myprovider.json [hunk 1]
```diff
@@ -6,34 +6,6 @@
     "name": "MyProvider"
   },
   "interactions": [
-    {
```
Comment (andykuszyk-form3) [inline internal/app/pacts/myconsumer-myprovider.json:9]: Why do we want to delete this interaction?

Comment (andykuszyk-form3) [reply internal/app/pacts/myconsumer-myprovider.json:9]: @sofib-form3 I think you added this commit, can you remember why this was necessary?

Comment (sofib-form3) [reply internal/app/pacts/myconsumer-myprovider.json:9]: yes, commit came from me - I cannot remember however why interaction was deleted, we should recheck it

Comment (sofib-form3) [reply internal/app/pacts/myconsumer-myprovider.json:9]: I've put it back on my fork - resync?

**Review by andykuszyk-form3 — COMMENTED** [review]
LGTM, but I think we need to document this new behaviour in the README.

File: internal/app/pacts/myconsumer-myprovider.json (hunk 1 above)
Comment (elliotwms-f3) [reply internal/app/pacts/myconsumer-myprovider.json:9]: This file is updated whenever the test are run, we have marked it as git ignored

**Review by elliotwms-f3 — APPROVED** [review]

**Pull request merged**